deploy/infrastructure/dev/us-east-2/versions.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 1.13.7"
```

Comment:
Typo?

Reply:
yup; I must have fat fingered it in previous PRs and not noticed :)

Reply:
That's defining the minimum version accepted to interact with terraform files here. It is set to the [lates version](https://www.terraform.io/downloads). What is the issue with using the latest version? 

Re change in this PR, I accidentally must have added a `3` in there, which I removed in this PR, i.e. the fat fingering.

Reply:
Oh! my mistake sorry I missed that terraform is already at 1.x. I was thinking 0.15 was the latest. Ignore me :)